cassandra/src/main/scala/geotrellis/spark/io/cassandra/CassandraLayerDeleter.scala

```diff
@@ -18,47 +18,48 @@ package geotrellis.spark.io.cassandra
 
 import geotrellis.spark.LayerId
 import geotrellis.spark.io._
+import geotrellis.util.LazyLogging
 
 import com.datastax.driver.core.querybuilder.QueryBuilder
 import com.datastax.driver.core.querybuilder.QueryBuilder.{eq => eqs}
 
 import scala.collection.JavaConversions._
 
-class CassandraLayerDeleter(val attributeStore: AttributeStore, instance: CassandraInstance) extends LayerDeleter[LayerId] {
+class CassandraLayerDeleter(val attributeStore: CassandraAttributeStore, instance: CassandraInstance) extends LazyLogging with LayerDeleter[LayerId] {
```

Comment:
Here can be used a generic `AttributeStore`, only `FileLayerDeleter` has a restricted attributeStore type here (but it's a local for-tests backend).